### ISSUE TITLE
SpaceNet : autopep8

### ISSUE THREAD

**Issue by lesteve** [nilearn/nilearn#733]
For new files, I think we should try to stive for PEP8 perfection ... running autopep8 on the SpaceNet files would be really appreciated.

```
❯ pep8 nilearn/decoding | grep -v searchlight
nilearn/decoding/objective_functions.py:301:29: E128 continuation line under-indented for visual indent
nilearn/decoding/proximal_operators.py:45:36: E701 multiple statements on one line (colon)
nilearn/decoding/proximal_operators.py:73:13: W503 line break before binary operator
nilearn/decoding/proximal_operators.py:77:15: E128 continuation line under-indented for visual indent
nilearn/decoding/proximal_operators.py:78:15: E128 continuation line under-indented for visual indent
nilearn/decoding/proximal_operators.py:169:33: W503 line break before binary operator
nilearn/decoding/proximal_operators.py:249:53: E202 whitespace before ')'
nilearn/decoding/proximal_operators.py:250:29: E128 continuation line under-indented for visual indent
nilearn/decoding/proximal_operators.py:295:35: E714 test for object identity should be 'is not'
nilearn/decoding/space_net.py:143:9: E128 continuation line under-indented for visual indent
nilearn/decoding/space_net.py:413:26: W503 line break before binary operator
nilearn/decoding/space_net.py:649:12: E714 test for object identity should be 'is not'
nilearn/decoding/space_net.py:679:12: E714 test for object identity should be 'is not'
nilearn/decoding/space_net.py:721:13: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:722:24: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:746:49: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:748:46: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:749:12: E714 test for object identity should be 'is not'
nilearn/decoding/space_net.py:749:33: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:750:29: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:751:13: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:757:17: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:765:27: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:766:13: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:769:13: E701 multiple statements on one line (colon)
nilearn/decoding/space_net.py:773:27: E701 multiple statements on one line (colon)
nilearn/decoding/space_net_solvers.py:57:19: W503 line break before binary operator
nilearn/decoding/space_net_solvers.py:86:13: W503 line break before binary operator
nilearn/decoding/space_net_solvers.py:159:48: E128 continuation line under-indented for visual indent
nilearn/decoding/space_net_solvers.py:189:44: E128 continuation line under-indented for visual indent
nilearn/decoding/space_net_solvers.py:210:22: W503 line break before binary operator
nilearn/decoding/space_net_solvers.py:223:13: W503 line break before binary operator
nilearn/decoding/space_net_solvers.py:238:31: E127 continuation line over-indented for visual indent
nilearn/decoding/space_net_solvers.py:239:31: E127 continuation line over-indented for visual indent
nilearn/decoding/space_net_solvers.py:278:57: E128 continuation line under-indented for visual indent
nilearn/decoding/space_net_solvers.py:298:27: E127 continuation line over-indented for visual indent
nilearn/decoding/space_net_solvers.py:299:27: E127 continuation line over-indented for visual indent
nilearn/decoding/space_net_solvers.py:340:63: E128 continuation line under-indented for visual indent
nilearn/decoding/space_net_solvers.py:537:1: W293 blank line contains whitespace
nilearn/decoding/tests/simulate_graph_net_data.py:18:5: E266 too many leading '#' for block comment
nilearn/decoding/tests/simulate_graph_net_data.py:28:5: E266 too many leading '#' for block comment
nilearn/decoding/tests/simulate_graph_net_data.py:37:5: E266 too many leading '#' for block comment
nilearn/decoding/tests/simulate_graph_net_data.py:49:5: E266 too many leading '#' for block comment
nilearn/decoding/tests/test_fista.py:6:36: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_fista.py:7:36: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_fista.py:8:36: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_fista.py:54:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_fista.py:55:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_fista.py:56:32: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_fista.py:57:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_fista.py:59:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_same_api.py:114:35: E127 continuation line over-indented for visual indent
nilearn/decoding/tests/test_same_api.py:115:35: E127 continuation line over-indented for visual indent
nilearn/decoding/tests/test_same_api.py:151:31: E127 continuation line over-indented for visual indent
nilearn/decoding/tests/test_same_api.py:156:23: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_same_api.py:157:23: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_same_api.py:159:25: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_same_api.py:160:25: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_same_api.py:187:25: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_same_api.py:188:25: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_sklearn_compatibility.py:16:30: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_sklearn_compatibility.py:27:25: E122 continuation line missing indentation or outdented
nilearn/decoding/tests/test_smooth_lasso.py:132:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_smooth_lasso.py:133:52: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_smooth_lasso.py:134:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_smooth_lasso.py:150:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_smooth_lasso.py:152:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_smooth_lasso.py:173:54: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_smooth_lasso.py:174:13: W503 line break before binary operator
nilearn/decoding/tests/test_smooth_lasso.py:175:56: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_smooth_lasso.py:193:13: W503 line break before binary operator
nilearn/decoding/tests/test_smooth_lasso.py:241:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_smooth_lasso.py:242:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_smooth_lasso.py:243:5: E731 do not assign a lambda expression, use a def
nilearn/decoding/tests/test_space_net.py:28:1: E402 module level import not at top of file
nilearn/decoding/tests/test_space_net.py:30:1: E402 module level import not at top of file
nilearn/decoding/tests/test_space_net.py:98:5: E129 visually indented line with same indent as next logical line
nilearn/decoding/tests/test_space_net.py:155:22: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_space_net.py:156:22: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_tv.py:22:30: E128 continuation line under-indented for visual indent
nilearn/decoding/tests/test_tv.py:23:30: E124 closing bracket does not match visual indentation
```

**Comment by dohmatob** [comment]
hmmm .. true. But I think we have a much bigger issue AFAICS: nilearn is
simply not pep8 compatible (in fact SpaceNet has increased the pep8
compatibility ... ;).

nilearn/input_data/base_masker.py:61:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:62:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:63:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:64:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:96:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:97:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:98:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:99:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:100:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:101:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:107:31: E127 continuation line
over-indented for visual indent
nilearn/input_data/base_masker.py:114:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:115:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:116:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:117:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:118:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:119:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:120:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:149:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:150:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:151:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:152:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:153:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:154:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:155:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:156:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:157:35: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:167:80: E501 line too long (91 > 79
characters)
nilearn/input_data/base_masker.py:206:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:207:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:208:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:209:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:210:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:211:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:212:33: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:213:27: E121 continuation line
indentation is not a multiple of four
nilearn/input_data/base_masker.py:222:80: E501 line too long (91 > 79
characters)
nilearn/input_data/base_masker.py:304:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:305:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:306:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:314:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:318:80: E501 line too long (81 > 79
characters)
nilearn/input_data/base_masker.py:326:13: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:327:13: E126 continuation line
over-indented for hanging indent
nilearn/input_data/base_masker.py:328:13: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:30:80: E501 line too long (87 > 79
characters)
nilearn/input_data/multi_nifti_masker.py:148:80: E501 line too long (91 >
79 characters)
nilearn/input_data/multi_nifti_masker.py:178:21: E128 continuation line
under-indented for visual indent
nilearn/input_data/multi_nifti_masker.py:181:25: E128 continuation line
under-indented for visual indent
nilearn/input_data/multi_nifti_masker.py:182:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:183:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:184:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:185:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:186:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:187:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:188:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/multi_nifti_masker.py:192:30: E128 continuation line
under-indented for visual indent
nilearn/input_data/multi_nifti_masker.py:220:80: E501 line too long (91 >
79 characters)
nilearn/input_data/multi_nifti_masker.py:235:21: E127 continuation line
over-indented for visual indent
nilearn/input_data/nifti_labels_masker.py:32:80: E501 line too long (87 >
79 characters)
nilearn/input_data/nifti_labels_masker.py:39:80: E501 line too long (87 >
79 characters)
nilearn/input_data/nifti_labels_masker.py:190:80: E501 line too long (91 >
79 characters)
nilearn/input_data/nifti_labels_masker.py:220:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_labels_masker.py:221:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_labels_masker.py:222:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_labels_masker.py:235:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_labels_masker.py:236:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:27:80: E501 line too long (87 > 79
characters)
nilearn/input_data/nifti_maps_masker.py:32:80: E501 line too long (87 > 79
characters)
nilearn/input_data/nifti_maps_masker.py:190:80: E501 line too long (91 > 79
characters)
nilearn/input_data/nifti_maps_masker.py:237:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:238:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:239:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:246:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:247:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:248:25: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_maps_masker.py:262:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_masker.py:18:1: W293 blank line contains whitespace
nilearn/input_data/nifti_masker.py:26:80: E501 line too long (87 > 79
characters)
nilearn/input_data/nifti_masker.py:151:80: E501 line too long (91 > 79
characters)
nilearn/input_data/nifti_masker.py:160:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_masker.py:161:29: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_masker.py:173:21: E128 continuation line
under-indented for visual indent
nilearn/input_data/nifti_masker.py:173:80: E501 line too long (89 > 79
characters)
nilearn/input_data/nifti_masker.py:206:80: E501 line too long (91 > 79
characters)
nilearn/input_data/nifti_spheres_masker.py:99:80: E501 line too long (87 >
79 characters)
nilearn/input_data/nifti_spheres_masker.py:223:80: E501 line too long (91 >
79 characters)
nilearn/input_data/nifti_spheres_masker.py:252:17: E126 continuation line
over-indented for hanging indent
nilearn/input_data/nifti_spheres_masker.py:256:38: E124 closing bracket
does not match visual indentation
nilearn/input_data/tests/test_base_masker.py:43:34: E124 closing bracket
does not match visual indentation
nilearn/input_data/tests/test_base_masker.py:47:33: E128 continuation line
under-indented for visual indent
nilearn/input_data/tests/test_base_masker.py:49:33: E128 continuation line
under-indented for visual indent
nilearn/input_data/tests/test_base_masker.py:63:26: E127 continuation line
over-indented for visual indent
nilearn/input_data/tests/test_multi_nifti_masker.py:123:17: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_labels_masker.py:23:21: E126
continuation line over-indented for hanging indent
nilearn/input_data/tests/test_nifti_labels_masker.py:104:1: E302 expected 2
blank lines, found 1
nilearn/input_data/tests/test_nifti_labels_masker.py:120:50: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_labels_masker.py:122:50: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_labels_masker.py:204:5: E303 too many
blank lines (2)
nilearn/input_data/tests/test_nifti_labels_masker.py:209:50: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_labels_masker.py:223:1: W391 blank line
at end of file
nilearn/input_data/tests/test_nifti_maps_masker.py:24:21: E126 continuation
line over-indented for hanging indent
nilearn/input_data/tests/test_nifti_maps_masker.py:46:19: E127 continuation
line over-indented for visual indent
nilearn/input_data/tests/test_nifti_maps_masker.py:113:50: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_maps_masker.py:134:50: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_maps_masker.py:136:50: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_maps_masker.py:139:19: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_maps_masker.py:213:19: E127
continuation line over-indented for visual indent
nilearn/input_data/tests/test_nifti_masker.py:126:17: E127 continuation
line over-indented for visual indent
nilearn/input_data/tests/test_nifti_masker.py:208:17: E127 continuation
line over-indented for visual indent
nilearn/input_data/tests/test_nifti_masker.py:233:80: E501 line too long
(84 > 79 characters)

Cheers!

On Wed, Jul 29, 2015 at 9:08 AM, Loïc Estève <notifications@github.com>
wrote:

> For new files, I think we should try to stive for PEP8 perfection ...
> running autopep8 on the SpaceNet files would be really appreciated.
>
> ❯ pep8 nilearn/decoding | grep -v searchlight
> nilearn/decoding/objective_functions.py:301:29: E128 continuation line under-indented for visual indent
> nilearn/decoding/proximal_operators.py:45:36: E701 multiple statements on one line (colon)
> nilearn/decoding/proximal_operators.py:73:13: W503 line break before binary operator
> nilearn/decoding/proximal_operators.py:77:15: E128 continuation line under-indented for visual indent
> nilearn/decoding/proximal_operators.py:78:15: E128 continuation line under-indented for visual indent
> nilearn/decoding/proximal_operators.py:169:33: W503 line break before binary operator
> nilearn/decoding/proximal_operators.py:249:53: E202 whitespace before ')'
> nilearn/decoding/proximal_operators.py:250:29: E128 continuation line under-indented for visual indent
> nilearn/decoding/proximal_operators.py:295:35: E714 test for object identity should be 'is not'
> nilearn/decoding/space_net.py:143:9: E128 continuation line under-indented for visual indent
> nilearn/decoding/space_net.py:413:26: W503 line break before binary operator
> nilearn/decoding/space_net.py:649:12: E714 test for object identity should be 'is not'
> nilearn/decoding/space_net.py:679:12: E714 test for object identity should be 'is not'
> nilearn/decoding/space_net.py:721:13: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:722:24: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:746:49: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:748:46: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:749:12: E714 test for object identity should be 'is not'
> nilearn/decoding/space_net.py:749:33: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:750:29: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:751:13: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:757:17: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:765:27: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:766:13: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:769:13: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net.py:773:27: E701 multiple statements on one line (colon)
> nilearn/decoding/space_net_solvers.py:57:19: W503 line break before binary operator
> nilearn/decoding/space_net_solvers.py:86:13: W503 line break before binary operator
> nilearn/decoding/space_net_solvers.py:159:48: E128 continuation line under-indented for visual indent
> nilearn/decoding/space_net_solvers.py:189:44: E128 continuation line under-indented for visual indent
> nilearn/decoding/space_net_solvers.py:210:22: W503 line break before binary operator
> nilearn/decoding/space_net_solvers.py:223:13: W503 line break before binary operator
> nilearn/decoding/space_net_solvers.py:238:31: E127 continuation line over-indented for visual indent
> nilearn/decoding/space_net_solvers.py:239:31: E127 continuation line over-indented for visual indent
> nilearn/decoding/space_net_solvers.py:278:57: E128 continuation line under-indented for visual indent
> nilearn/decoding/space_net_solvers.py:298:27: E127 continuation line over-indented for visual indent
> nilearn/decoding/space_net_solvers.py:299:27: E127 continuation line over-indented for visual indent
> nilearn/decoding/space_net_solvers.py:340:63: E128 continuation line under-indented for visual indent
> nilearn/decoding/space_net_solvers.py:537:1: W293 blank line contains whitespace
> nilearn/decoding/tests/simulate_graph_net_data.py:18:5: E266 too many leading '#' for block comment
> nilearn/decoding/tests/simulate_graph_net_data.py:28:5: E266 too many leading '#' for block comment
> nilearn/decoding/tests/simulate_graph_net_data.py:37:5: E266 too many leading '#' for block comment
> nilearn/decoding/tests/simulate_graph_net_data.py:49:5: E266 too many leading '#' for block comment
> nilearn/decoding/tests/test_fista.py:6:36: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_fista.py:7:36: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_fista.py:8:36: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_fista.py:54:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_fista.py:55:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_fista.py:56:32: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_fista.py:57:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_fista.py:59:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_same_api.py:114:35: E127 continuation line over-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:115:35: E127 continuation line over-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:151:31: E127 continuation line over-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:156:23: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:157:23: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:159:25: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:160:25: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:187:25: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_same_api.py:188:25: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_sklearn_compatibility.py:16:30: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_sklearn_compatibility.py:27:25: E122 continuation line missing indentation or outdented
> nilearn/decoding/tests/test_smooth_lasso.py:132:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_smooth_lasso.py:133:52: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_smooth_lasso.py:134:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_smooth_lasso.py:150:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_smooth_lasso.py:152:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_smooth_lasso.py:173:54: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_smooth_lasso.py:174:13: W503 line break before binary operator
> nilearn/decoding/tests/test_smooth_lasso.py:175:56: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_smooth_lasso.py:193:13: W503 line break before binary operator
> nilearn/decoding/tests/test_smooth_lasso.py:241:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_smooth_lasso.py:242:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_smooth_lasso.py:243:5: E731 do not assign a lambda expression, use a def
> nilearn/decoding/tests/test_space_net.py:28:1: E402 module level import not at top of file
> nilearn/decoding/tests/test_space_net.py:30:1: E402 module level import not at top of file
> nilearn/decoding/tests/test_space_net.py:98:5: E129 visually indented line with same indent as next logical line
> nilearn/decoding/tests/test_space_net.py:155:22: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_space_net.py:156:22: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_tv.py:22:30: E128 continuation line under-indented for visual indent
> nilearn/decoding/tests/test_tv.py:23:30: E124 closing bracket does not match visual indentation
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/issues/733>.
>



-- 
DED

**Comment by lesteve** [comment]
> But I think we have a much bigger issue AFAICS: nilearn is simply not pep8 compatible

Granted, I am just saying that for new files, there is no reason not to run autopep8 on them ...

**Comment by dohmatob** [comment]
OK, then I'm closing the issue.

**Comment by lesteve** [comment]
> OK, then I'm closing the issue.

Apparently I wasn't explicit enough:

>> For new files, I think we should try to stive for PEP8 perfection ... running autopep8 on the SpaceNet files would be really appreciated.

meant would you be kind enough you run autopep8 on the new files that SpaceNet introduced and open a PR?

@AlexandreAbraham has already fixed the tests files in #735, so you only need to do it for the remaining files.

**Comment by lesteve** [comment]
Fixed by #739.
